src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs

```diff
@@ -322,6 +351,36 @@ public override void RequestRefresh()
             }
         }
 
+        internal static void TrackRetrievalDuration(TimeSpan duration)
+        {
+            _telemetryClient.LogOperationDuration(
+                IdentityModelTelemetryUtil.ClientVer,
+                (long)duration.TotalMilliseconds);
+        }
+
+        internal static void TrackRetrievalDuration(TimeSpan duration, string exception)
```

Comment:
should this be exception => exceptionType, same as a few lines down?

Reply:
Yes, I will fix that. Thanks!